pkgs/tools/misc/ntfy-sh/default.nix

```
@@ -6,7 +6,9 @@ let
   nodeDependencies = (import ./node-composition.nix {
     inherit pkgs nodejs;
     inherit (stdenv.hostPlatform) system;
-  }).nodeDependencies;
+  }).nodeDependencies.override {
+    buildInputs = with pkgs; [ pkgconfig pixman cairo pango ];
```

`pkgs.pkgconfig` does not exist. I wonder why this does not fail in CI?

```suggestion
    buildInputs = with pkgs; [ pkg-config pixman cairo pango ];
```

---

`pkgconfig` does exist, and `pkg-config` is just an alias for it

---

`nixpkgs-review pr 230188` output:
```
error: undefined variable 'pkgconfig'

       at /home/teutat3s/.cache/nixpkgs-review/pr-230188/nixpkgs/pkgs/tools/misc/ntfy-sh/default.nix:10:32:

            9|   }).nodeDependencies.override {
           10|     buildInputs = with pkgs; [ pkgconfig pixman cairo pango ];
             |                                ^
           11|   };
(use '--show-trace' to show detailed location information)
```

---

> `pkgconfig` does exist, and `pkg-config` is just an alias for it

It's actually the opposite. The OfBorg `ofborg-eval-package-list-no-aliases` check is supposed to be catching these but it only seems to work when the alias is a package arg, not when you try to fish it out of `pkgs`

---

That's really weird. I tested the build successfully before submitting, and also checked the packages and alias. I'll try again tonight and update the PR if necessary.

---

Okay, I figured it out and you're right. I tried to do the right thing, but in doing so I misread the aliases file. It's fixed now, and I also removed the use of `pkgs`.